### ISSUE TITLE
Work with <String, String>{} returns exception type 'String' is not a subtype of type '$Value' of 'key'

### ISSUE THREAD

**Issue by al-tush** [ethanblake4/dart_eval#186]
Executing code
```
final res = <String, String>{};
res['1'] = '2';
```
returns exception
```
type 'String' is not a subtype of type '$Value' of 'key'

The relevant error-causing widget was: 
  MaterialApp MaterialApp:file:///C:/Projects-test/eval1/lib/main.dart:15:12
When the exception was thrown, this was the stack: 
#0      Runtime.execute (package:dart_eval/src/eval/runtime/runtime.dart:830:7)
#1      Runtime.executeLib (package:dart_eval/src/eval/runtime/runtime.dart:810:12)
#2      eval (package:dart_eval/src/eval/eval.dart:58:26)
```

**dart_eval**: 0.7.5

This is a continuation of #180

**Comment by ethanblake4** [comment]
Fixed in v0.7.6

**Comment by al-tush** [comment]
It works perfect now. Thank you!
